src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -29,9 +28,18 @@ private int Run(string[] args)
             var errs = new ArrayList();
             bool force = false;
             bool proxyOnly = false;
+            bool caseSensitive = false;
 
             try
             {
+                if (args.Length > 0)
+                {
+                    if (args.Where(s => s.IndexOf("casesensitive", StringComparison.OrdinalIgnoreCase) >= 0).Count() > 0)
```

Comment:
It's simpler and more efficient to use `Any` instead of `Where` and `Count`, e.g.:

```c#
if (args.Any(s => s.IndexOf("casesensitive", StringComparison.OrdinalIgnoreCase) >= 0))
```